tests/python/conftest.py

```diff
@@ -78,7 +78,7 @@ def mock_host_session(mock_host_interface, mock_logger):
     Fixture for a `HostSesssion`, configured with a mock_host_interface,
     that forwards method calls to an internal public `mock.Mock` instance.
     """
-    return MockHostSession(Host(mock_host_interface), mock_logger)
+    return HostSession(Host(mock_host_interface), mock_logger)
```

Comment:
[consider][minor] `test_hostsession.py` duplicates this fixture somewhat (though it splits out `a_host` to its own fixture)

Reply:
Switched to using the shared fixture(s) [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/541/commits/fc13ea7c654728678dd1c5ad522ffe65f8bfbf67).

Reply:
Looks good, just the pylint failure due to this removal (noticed via the github annotations :smile: )

Reply:
^ ah OK, fixed in [75dc42b](https://github.com/OpenAssetIO/OpenAssetIO/pull/541/commits/75dc42b18c711a7e36324acfff0156f0ae65c6d4)